gammapy/modeling/models/spatial.py

```diff
@@ -156,8 +171,11 @@ def plot(self, ax=None, geom=None, **kwargs):
 
         data = self.evaluate_geom(geom)
         m = Map.from_geom(geom, data=data.value, unit=data.unit)
-        _, ax, _ = m.plot(ax=ax, **kwargs)
 
+        if len(m.data.shape) > 2:
```

Comment:
Maybe for consistency we just copy the `Map` pattern here? So expose the interactive plotting using a separate `.plot_interactive()` method? I can see that merging results in a more compact API, but I think consistency is more important here...